src/assets/new/icons/shopping-cart.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" standalone="yes"?>
```

Comment:
this the second of same type you downloaded

Reply:
removed